src/Appwrite/Utopia/Response/Model/File.php

```diff
@@ -13,37 +13,44 @@ public function __construct()
             ->addRule('$id', [
                 'type' => self::TYPE_STRING,
                 'description' => 'File ID.',
+                'default' => '',
                 'example' => '5e5ea5c16897e',
             ])
             ->addRule('$permissions', [
                 'type' => Response::MODEL_PERMISSIONS,
                 'description' => 'File permissions.',
+                'default' => '',
                 'example' => new \stdClass,
                 'array' => false,
             ])
             ->addRule('name', [
                 'type' => self::TYPE_STRING,
                 'description' => 'File name.',
+                'default' => '',
                 'example' => 'Pink.png',
             ])
             ->addRule('dateCreated', [
                 'type' => self::TYPE_INTEGER,
                 'description' => 'File creation date in Unix timestamp.',
+                'default' => '',
                 'example' => 1592981250,
             ])
             ->addRule('signature', [
                 'type' => self::TYPE_STRING,
                 'description' => 'File MD5 signature.',
+                'default' => '',
                 'example' => '5d529fd02b544198ae075bd57c1762bb',
             ])
             ->addRule('mimeType', [
                 'type' => self::TYPE_STRING,
                 'description' => 'File mime type.',
+                'default' => '',
                 'example' => 'image/png',
             ])
             ->addRule('sizeOriginal', [
                 'type' => self::TYPE_INTEGER,
                 'description' => 'File original size in bytes.',
+                'default' => '',
```

Comment:
Must be `0` 🙂

Reply:
Fixed...

Hehe you got to love a good code review :)